docs/pages/access-controls/guides/locking.mdx

```diff
@@ -80,6 +81,13 @@ with one of the following options:
   # Created a lock with name "dc7cee9d-fe5e-4534-a90d-db770f0234a1".
   ```
   </TabItem>
+  <TabItem label="Access request">
+  All connections using elevated privileges from the matching access request will be locked.
```

Comment:
I thought locks didn't apply to your connections to leaf clusters, or am I misremembering that?